data/human/pirate jobs.txt

```diff
@@ -3396,3 +3396,21 @@ mission "Republic Assassination [4]"
 		"reputation: Republic" -= 25
 		"reputation: Navy (Oathkeeper)" -= 25
 		"reputation: Pirate" += 10
+
+mission "Slave revolt"
+	landing
+	repeat
+	to offer
+		random < 10
+	on offer
+		require "Slave" 2
+		conversation
+			`	You are flicking through your diary in your cabin, when you hear your door start to open.`
+			`	You look up, one of your slaves is standing in the doorway... holding a large metal bar.`
+			choice
+				`    Grab for a weapon.`
+					goto grab
+			label "grab"
+			`	You immediately try to grab the antique sword hanging on the wall that your grandfather gave you before you left to be a space pilot.`
+			`	As you reach for it the slave rushes in and brings the bar down on your head. All goes black...`
+				die
```

Comment:
The moral of this story boils down to "owning slaves is bad because they fight back sometimes". Set up the story more, give us details about how First Last is treating these people that causes them to revolt - hell, give the player a warning event where they get the chance to free their slaves before they free themselves.

Reply:
I am still greatly unsatisfied with this text; it's a bit short, and doesn't grant the player any initiative to try and avoid their fate (a choice of grabbing your gun, or attempting to dodge, which either way ends in your demise, would be good), and just calls your assailant "the slave" - characterize them! This person is the last thing First Last sees before they die, and you don't even tell us what they look like! Do they have a scowl on their face? Grim determination? Desperation? And where did they get the metal bar? Surely First Last would try to keep their victims on a tight leash, how could they have obtained and then kept hidden a large metal bar?